cime_config/testdefs/testlist_clm.xml

```diff
@@ -1851,6 +1851,22 @@
       <option name="wallclock">00:40:00</option>
     </options>
   </test>
+  <test name="ERS_Lm12" grid="f10_f10_mg37" compset="I2000Clm50Fates" testmods="clm/FatesColdDef">
```

Comment:
Lm13?  That should ensure trimming is being restarted correctly right?

Reply:
`Lm12` will capture the `trim_canopy` call and the necessary restart.  That said, @billsacks and @ekluzek did agree that `Lm13` would be good to help capture ctsm year rollover affects.

Reply:
Yes, exactly.